pom.xml

```diff
@@ -86,5 +86,7 @@
         <module>part-4-routes</module>
         <module>part-5-extensions</module>
         <module>part-6-eips</module>
```

Comment:
Could you please test the maven dependencies pre-download section ? I think this PR will break it.